pkg/sparrow/controller.go

```diff
@@ -83,13 +80,10 @@ func (cc *ChecksController) Run(ctx context.Context) error {
 // Shutdown shuts down the ChecksController.
 func (cc *ChecksController) Shutdown(ctx context.Context) (err error) {
```

Comment:
You may also remove the error return here, either in this PR or in the later one for #123

Reply:
Later